cnames_active.js

```diff
@@ -507,6 +507,7 @@ var cnames_active = {
   "fuck": "gerd2002.github.io/fuck.js",
   "funbook": "egoist.github.io/funbook",
   "funfuncs": "floffah.github.io/funfuncs",
+  "fusuhub": " fusuhub.github.io",
```

Comment:
```suggestion
  "fusuhub": "fusuhub.github.io",
```

The CNAME present on your site goes to `fucao.js.org` not `fusuhub.js.org` which is what you are requesting.

Reply:
Corrected! Thank you~~